README.md

```diff
@@ -969,7 +985,9 @@ Front. Neurosci. (2016), 10:391. [PDF](https://www.neuromorphic-vision.com/publi
 - <a name="Han20cvpr"></a>Han, J., Zhou, C., Duan, P., Tang, Y., Xu, C., Xu, C., Huang, T., Shi, B.,  
 *[Neuromorphic Camera Guided High Dynamic Range Imaging](http://openaccess.thecvf.com/content_CVPR_2020/html/Han_Neuromorphic_Camera_Guided_High_Dynamic_Range_Imaging_CVPR_2020_paper.html)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2020. [PDF](http://openaccess.thecvf.com/content_CVPR_2020/papers/Han_Neuromorphic_Camera_Guided_High_Dynamic_Range_Imaging_CVPR_2020_paper.pdf), [Suppl. Mat.](http://openaccess.thecvf.com/content_CVPR_2020/supplemental/Han_Neuromorphic_Camera_Guided_CVPR_2020_supplemental.pdf)
-
+- <a name="messikommer22cvprw"></a>N. Messikommer*, S. Georgoulis*, D. Gehrig, S. Tulyakov, J. Erbach, A. Bochicchio, Y. Li, D. Scaramuzza,  
+*[Multi-Bracket High Dynamic Range Imaging with Event Cameras](https://arxiv.org/abs/2203.06622)*,  
+IEEE Conference on Computer Vision and Pattern Recognition Workshop (CVPRW), 2022. [PDF](https://rpg.ifi.uzh.ch/docs/CVPRW22_Messikommer.pdf), [YouTube](https://youtu.be/fw9-gNg6cM8).  
```

Comment:
This conference name formatting is not consistent with all other CVPRWs. Please be consistent, to facilitate the search with Ctrl+F.